pitop/processing/algorithms/faces/face_detector_haarcascade.py

```diff
@@ -0,0 +1,121 @@
+import os
+import cv2
+from pitop.processing.utils.vision_functions import (
+    find_largest_rectangle,
+    resize,
+    center_reposition
+)
+from pitop.core import ImageFunctions
+
+# directory where calibration output pickle file is located
+classifier_dir = 'classifiers'
+script_dir = os.path.dirname(os.path.realpath(__file__))
+abs_file_path = os.path.join(script_dir, classifier_dir)
+
+# Filename for classifier
+cascade_model_classifier = 'haarcascade_frontalface_default.xml'
+
+
+class DotDict(dict):
```

Comment:
lets put this into a "utils" file maybe in `pitop.core` since it's used in 2 places now

Reply:
yeah agree